src/makeRobot.js

```diff
@@ -38,7 +38,68 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const Robot = {
```

Comment:
don't name regular variables with capital letter
```suggestion
  const robot = {
```